### PR TITLE
feat(textarea): Adding textarea component

### PR DESCRIPTION
**Pull request by benjaminParisel**
Covers [UID-557](https://bonitasoft.atlassian.net/browse/UID-557)

### CODE REVIEW

File: components/uid-textarea/package.json [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    "src/UidTextarea.ts",
+    "src/uid-textarea.ts",
+    "test"
+  ],
+  "exports": {
+    ".": "./dist/src/index.js",
+    "./uid-textarea.js": "./dist/src/uid-textarea.js"
+  },
+  "scripts": {
+	"gen-widget": "cwb gen-widget --propertiesFile uidTextarea.json --webComponentBundle lib/uid-textarea.es5.min.js --outputDir uid_widget",
```
Comment (jeromecambon) [inline components/uid-textarea/package.json:21]: Remove this (duplicated), it has been added by widget duplicate. It has been fixed now, please upate your sdk.

File: components/uid-textarea/README.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
+# \<uid-textarea>
+![npmVersion](https://img.shields.io/npm/v/uid-textarea?color=blue&style=plastic)
+
+Simple input (with label) web component.
```
Comment (jeromecambon) [inline components/uid-textarea/README.md:4]: to update

File: components/uid-textarea/README.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  import 'uid-textarea/dist/uid-textarea.bundle.min.js';
+</script>
+
+<uid-textarea></uid-textarea>
+```
+
+## Attributes
+
+| Attribute        | Type      | Default | Possible values    |
+|------------------|-----------|---------|--------------------|
```
Comment (jeromecambon) [inline components/uid-textarea/README.md:29]: update the property list

File: components/uid-textarea/package.json [hunk 4]
```diff
@@ -0,0 +1,135 @@
+{
+  "name": "uid-textarea",
+  "description": "An input with label",
+  "homepage": "https://github.com/bonitasoft/web-components/components/uid-textarea#readme",
+  "license": "GPLv2",
+  "version": "1.0.1",
```
Comment (jeromecambon) [inline components/uid-textarea/package.json:6]: version 1.0.0

File: components/uid-textarea/package.json [hunk 5]
```diff
@@ -0,0 +1,135 @@
+{
+  "name": "uid-textarea",
+  "description": "An input with label",
```
Comment (jeromecambon) [inline components/uid-textarea/package.json:3]: to update

**Review by jeromecambon — APPROVED** [review]
LGTM,
- some update missing here and there
- 'required' story does not work in Storybook

**Comment by benjaminParisel** [comment]
@jeromecambon `required` stories don't work on chrome, it's an issue from ::part :ok_hand:

**Comment by jeromecambon** [comment]
> @jeromecambon `required` stories don't work on chrome, it's an issue from ::part 👌

Ha? I think it worked for uid-input, isn't it?

**Pull request merged**